test-configs.yaml

```diff
@@ -1030,11 +1020,76 @@ device_types:
     filters:
       - whitelist: {defconfig: ['defconfig']}
 
+  r8a7743-iwg20d-q7:
```

Comment:
This one doesn't seem happy either:
https://lava.ciplatform.org/scheduler/device_type/r8a7743-iwg20d-q7?dt_length=25&dt_search=kernelci#dt_